padle/padle.py

```diff
@@ -0,0 +1,485 @@
+from tsutils.tsubaki.links import MonsterImage, MonsterLink
+from tsutils.tsubaki.monster_header import MonsterHeader
+
+from tsutils.cogs.userpreferences import get_user_preference
+from tsutils.helper_functions import conditional_iterator
+from tsutils.user_interaction import get_user_confirmation
+from tsutils.emoji import NO_EMOJI, SendableEmoji, YES_EMOJI
+
+from discordmenu.embed.components import EmbedThumbnail, EmbedMain
+from discordmenu.embed.view import EmbedView
+from redbot.core.utils.menus import (
+    menu,
+    next_page,
+    prev_page,
+    start_adding_reactions
+)
+
+
+class PADle(commands.Cog):
+    """PADle"""
+
+    def __init__(self, bot, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.bot = bot
+        self.config = Config.get_conf(self, identifier=123456789)
+        self.config.register_user(guesses=[], start=False, done=False, score=[], editID="", channelID="", embedText=[])
+        self.config.register_global(padleToday=1, storedDay=0, numDays=0, subs=[], allScores=[], saveDailyScores=[])
+
+        self._daily_padle_loop = bot.loop.create_task(self.generatePadle())
+
+    async def get_dbcog(self) -> "DBCog":
+        dbcog = self.bot.get_cog("DBCog")
+        if dbcog is None:
+            raise ValueError("DBCog cog is not loaded")
+        await dbcog.wait_until_ready()
+        return dbcog
+
+    @commands.group()
+    async def padle(self, ctx):
+        """Commands pertaining to PADle"""
+
+    @padle.command()
+    async def help(self, ctx):
+        """Instructions for PADle"""
+        await ctx.send("- PADle is similar to Wordle, except for PAD cards.\n"
+                       "- You have infinite tries to guess the hidden PAD card (chosen from a list "
+                       "of more-well known monsters). Everyone is trying to guess the same PAD card. "
+                       "With each guess, you are given feedback as to how similar the two cards are, "
+                       "including comparing the awakenings, rarity, typings, attributes, and monster "
+                       "point sell value.\n- A new PADle is available every day. Use `^padle start` to begin!")
+
+    @padle.command(aliases=["subscribe", "togglesub"])
+    async def togglesubscribe(self, ctx):
+        """Toggles daily notifications of new PADles"""
+        subbedUsers = await self.config.subs()
+        if ctx.author.id in subbedUsers:
+            await ctx.send("You will no longer recieve notifications of new PADles.")
+            async with self.config.subs() as subs:
+                subs.remove(ctx.author.id)
+            return
+        await ctx.send("You will now recieve notifications of new PADles.")
+        async with self.config.subs() as subs:
+            subs.append(ctx.author.id)
+
+    @padle.command()
+    async def start(self, ctx):
+        """Start a game of PADle"""
+        if(ctx.guild is not None):
+            await ctx.send("You can only play PADle in DMs!")
```

Comment:
It would be cool to say like, Start game now? and if you confirm then send a dm to the user beginning the game (talk to aradia if you need help actually implementing this)